config-default.yml

```diff
@@ -251,7 +256,7 @@ guild:
         duck_pond:                      637821475327311927
         dev_log:                        680501655111729222
         python_news:    &PYNEWS_WEBHOOK 704381182279942324
-
+        incidents_archive:              720671599790915702
```

Comment:
Kaizened in 54891b4, also indented the `&PYNEWS_WEBHOOK` further since I was already making changes.

The config files (both the yaml and the `.py`) could use some more refactoring, I noticed that the spacing between classes is inconsistent (1 or 2 empty lines), and some other constants are probably not sorted properly. But I consider those beyond the scope of this PR.